Documentation/Upgrade/UpdateReferenceIndex/Index.rst

```diff
@@ -24,6 +24,10 @@ In a non-composer installation execute::
 
    php ./typo3/sysext/core/bin/typo3 referenceindex:update
 
+.. tip::
+
+   Use `referenceindex:update 2>> /dev/null` to suppress the output of the symfony progressbar to stderr
```

Comment:
Why not `1>&2` and drop `/dev/null`? The text written here suggests to redirect `stdout` to `stderr`, not dropping it completely. That would be achieved with `2>&1 > /dev/null`.

Reply:
Sadly the symfony progressbar uses `stderr` by default (see discussion at https://github.com/symfony/symfony/issues/18744), so if you update the reference index by a cronjob, the output of the progressbar will be logged to the cronjob output. When you have huge tables with hundred thousands of record, the cron output will be flooded like shown below.

```
Update index of table sys_file_metadata
---------------------------------------

      0/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%
    689/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%
   1739/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%
   2072/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%
   2196/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%
   2476/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%
   3147/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   1%
   4099/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   1%
   5064/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   1%
   6019/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   2%
   6932/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   2%
   7901/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   2%
   9151/295520 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   3%
   .....
```

To avoid this, I recommend to redirect the output of `stderr` to `/dev/null` as shown in the example. Doing so, the output of stdout (notifications/possible errors from the reference index update task) will still be logged.

Reply:
OK, got that. And the double `>` was intentional or a RST thingy?

Reply:
The double `>` was a mistake. I updated the PR

Reply:
OK, thanks. :-)